app/assets/javascripts/components/articles/article.jsx

```diff
@@ -44,6 +44,7 @@ const Article = createReactClass({
   render() {
     const ratingClass = `rating ${this.props.article.rating}`;
     const ratingMobileClass = `${ratingClass} tablet-only`;
+    const isArticle = project !== 'wikidata';
```

Comment:
Where does `project` come from?

Reply:
From the article (props).
https://github.com/1v4n4/WikiEduDashboard/blob/1b6b98351d3f2c06e753d3b402b320006837eb00/app/assets/javascripts/components/articles/article.jsx#L72-L74
That one is ok, right? I'll change all variables to isWikipedia.